scripts/cmake/includedCMakeLists.txt

```diff
@@ -3,21 +3,17 @@
 # the top level of CIME or the top level of ACME. The required
 # CIME_REL_DIR variable is set in CIME to be "." and in ACME
 # to be "cime".  Except for this misdirection, this is meant
-# to be the full-functioning top-level CMakeLists.txt file 
+# to be the full-functioning top-level CMakeLists.txt file
```

Comment:
Now using this file as a template and copying it to the case bld directory in the case.setup phase.